platform/branding-api/pom.xml

```diff
@@ -21,9 +21,8 @@
         <artifactId>platform</artifactId>
         <version>2.11.0-SNAPSHOT</version>
     </parent>
-    <groupId>ddf.distribution</groupId>
     <artifactId>branding-api</artifactId>
```

Comment:
I think it would be a good idea to update the coordinates to reflect the new location.  Downstream projects should get the changes from the DDF distribution.